memory/memory_darwin.go

```diff
@@ -22,3 +23,26 @@ func getMemoryInfo() (memoryInfo map[string]string, err error) {
 
 	return
 }
+
+func getMemoryInfoByte() (uint64, uint64, error) {
+	memInfo, err := getMemoryInfo()
+	var mem, swap uint64
+
+	// mem is already in bytes but `swap_total` use the format "5120,00M"
+	if v, ok := memInfo["swap_total"]; ok {
+		idx := strings.IndexAny(v, ",.") // depending on the local either a comma or dot is used
+		swapTotal, e := strconv.ParseUint(v[0:idx], 10, 64)
+		if e == nil {
+			swap = swapTotal * 1024 * 1024 // swapTotal is in mb
+		}
+	}
+
+	if v, ok := memInfo["total"]; ok {
+		t, e := strconv.ParseUint(v, 10, 64)
+		if e == nil {
+			mem = t // mem is returned in bytes
+		}
+	}
```

Comment:
Errors in parsing are ignored. I would assume that this is unintended. We probably should at least print that there was an issue.